docs/maven-plugin.md

```diff
@@ -150,6 +150,7 @@ If you want to use a custom set of rules you will have to set the configuration
       <checkstyleRuleset>build-tools/checkstyle/binding.xml</checkstyleRuleset>
       <checkstyleFilter>build-tools/checkstyle/suppressions.xml</checkstyleFilter>
       <pmdRuleset>build-tools/pmd/binding.xml</pmdRuleset>
+      <pmdFilter>build-tools/pmd/suppressions.properties</pmdFilter>

```

Comment:
Could you take a look at the "Maven plugin goals and parameters" section in the same file.

In the section "pmd" you can add the new property with a description.